### PR TITLE
🌐 Adds Bahasa Indonesia

### LINKED ISSUE #110: New language support suggestion : Bahasa Indonesia
**I would like to ...**

- [x] offer my help to translate this cheat sheet in another language
- [ ] ask if someone can translate this cheat sheet in another language

**What is the language you would see this cheat sheet translate ?**
Bahasa Indonesia

### PR DESCRIPTION
**Pull request by HanzCEO**
Resolves #110 

Let me know how you think about it, thank you!

### CODE REVIEW

File: README-id.md [hunk 1]
```diff
@@ -0,0 +1,327 @@
...
+  </a>
+</p>
+
+<h1 align="center">Panduan promosi proyek Open source</h1>
+
+> Panduan sederhana ini merangkum langkah-langkah utama yang baik dilakukan untuk memasarkan proyek open source untuk hasil yang memuaskan. Anda dapat menekan setiap poin untuk penjelasan yang lebih lengkap.
+
+Berbagai pilihan bahasa lainnya:
+
+- [العربية](./README-ar.md)
```
Comment (nlepage) [inline README-id.md:17]: Avoids the list being displayed in right to left:

```suggestion
- &ltr;[العربية](./README-ar.md)
```

**Comment by nlepage** [comment]
Hi @HanzCEO 
Thanks for your contribution!
Do you know anyone who could review the pull request?

**Comment by nlepage** [comment]
Also you may want to add the new README file in the [files list in .all-contributorsrc](https://github.com/zenika-open-source/promote-open-source-project/blob/2a04244c491b4656f76d23266e0aea4658e67067/.all-contributorsrc#L2).

**Comment by HanzCEO** [comment]
> Hi @HanzCEO Thanks for your contribution! Do you know anyone who could review the pull request?
https://github.com/zenika-open-source/promote-open-source-project/pull/115#issuecomment-999416134

I will try to invite somebody from Indonesian programming communities, wish me luck!

**Review by nlepage — APPROVED** [review]

**Comment by nlepage** [comment]
> I will try to invite somebody from Indonesian programming communities, wish me luck!

Hey @HanzCEO
I don't want to keep this PR opened for too long, is it OK with you if we merge it now ?
If any correction is needed on your transaltion, we can always open another PR later.

**Comment by HanzCEO** [comment]
I'm fine with the conclusion, I think that would create opportunity for
others to contribute too.

**Pull request merged**